tools/pq/README.md

```diff
@@ -0,0 +1,32 @@
+# pq
+
+pq is a CLI tool for working with delayed messages on poison queues.
+
+For now, it supports the PostgreSQL Pub/Sub implementation.
+
+## Install
+
+```bash
+go install github.com/ThreeDotsLabs/watermill/tools/pq@latest
+```
+
+## Usage
+
+Set the `DATABASE_URL` environment variable to your PostgreSQL connection string.
```

Comment:
it would be nice to provide example one - they are 10 different formats and it's useful to have it

Reply:
cc91ad38bf2c9219e20042be5ea46335c02e1c06